requirements.txt

```diff
@@ -113,3 +113,4 @@ wcwidth==0.1.8; python_version >= "3.4" \
 werkzeug==1.0.0 \
     --hash=sha256:6dc65cf9091cf750012f56f2cad759fa9e879f511b5ff8685e456b4e3bf90d16 \
     --hash=sha256:169ba8a33788476292d04186ab33b01d6add475033dfc07215e6d219cc077096
+requests==2.23.0; python_version >= "3.4"
```

Comment:
This project is using Poetry the way to add a new dep is

1 -add to poetry
```
poetry add requests
```
or include manually on pyproject.toml dependencies list.

then run `poetry export -f requirements.txt -o requirements.txt` to export it to the requirements.txt with the proper hashes.